src/mscorlib/src/System/Text/Normalization.cs

```diff
@@ -323,5 +324,27 @@ internal static String Normalize(String strInput, NormalizationForm normForm)
         [DllImport(JitHelpers.QCall, CharSet = CharSet.Unicode)]
         unsafe private static extern void nativeNormalizationInitNormalization(
             NormalizationForm normForm, byte* pTableData);
+#else
+        unsafe private static int nativeNormalizationNormalizeString(
```

Comment:
FYI: @joperezr as you were looking at some normalization issues as part of your work.